dev/App.tsx

```diff
@@ -18,7 +19,7 @@ export default class App extends Component {
     }, 2000);
   }
 
-  handleQuickButtonClicked = (e) => {
+  handleQuickButtonClicked = (e: any) => {
```

Comment:
If this is a react event, there's a type for react events, I don't remember exactly which one, maybe `React.FormEvent<Element>`. If it's a string, type it